client/lib/products-values/constants.js

```diff
@@ -116,7 +116,7 @@ export const JETPACK_PRODUCTS = [
 		optionDescriptions: {
 			...JETPACK_BACKUP_PRODUCT_DESCRIPTIONS,
 		},
-		optionsLabel: translate( 'Backup Options' ),
+		optionsLabel: translate( 'Backup Options:' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.